src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -44,121 +61,319 @@ export function formatDate(date) {
+      contents.description = (
+        <p>You have opted to not continue your appeal. If this information is incorrect, please
+        contact your Veteran Service Organization or representative for more information.</p>
+      );
+      break;
+    case STATUS_TYPES.ftr:
+      contents.title = 'Your appeal was closed';
+      contents.description = (
+        <p>You did not take an action VA requested in order to continue your appeal. If this
+        information is incorrect, or if you want to reopen your appeal, please contact your Veteran
+        Service Organization or representative for more information.</p>
+      );
+      break;
+    case STATUS_TYPES.ramp:
+      contents.title = 'You opted in to the Rapid Appeals Modernization Program (RAMP)';
+      contents.description = (
         <div>
-          For issues that are allowed, you will receive compensation. For more information, please
-          contact your VSO or representative.
+          <p>
+            You chose to participate in the new supplemental claim or higher-level review lanes.
+            This does not mean that your appeal has been closed. If this information is incorrect,
+            please contact your Veteran Service Organization or representative as soon as possible.
+          </p>
+          <p>
+            At this time, Vets.gov is not able to provide information about appeals that are part
+            of RAMP.
+          </p>
         </div>
-      </div>
-    );
-  } else if (type === pendingForm9) {
-    contents.title = 'Review your Statement of the Case';
-    contents.description = (
-      <div>
-        <p>
-          The Veterans Benefits Administration (VBA) sent you a Statement of the Case on November
-          28, 2017. The Statement of the Case document explains the reasons why the Regional Office
-          could not fully grant your appeal. If you don’t agree with the Statement of the Case, you
-          can bring your appeal to the Board of Veterans’ Appeals. To do this you must complete and
-          return a Form 9 within 60 days.
-        </p>
-        <p>
-          <em>Note:</em> If you send more evidence with the Form 9, it will be reviewed by the
-          Regional Office and will likely delay a decision on your appeal. If you have new
-          evidence, it’s best to send it when the Board of Veterans Appeals is reviewing your
-          case.
-        </p>
-      </div>
-    );
-  } else if (type === onDocket) {
-    contents.title = 'Waiting to be assigned to a judge';
-    contents.description = (
-      <p>Your appeal is at the Board of Veterans’ Appeals waiting to be
-      assigned to a Veterans Law Judge. Staff at the Board are making sure that your case is
-      complete, accurate, and ready to be decided by a judge. If you have evidence that isn’t
-      already included in your case, now is a good time to send it to the Board.</p>);
-  } else {
-    contents.title = 'Current Status Unknown';
```

Comment:
I'm going to create a separate issues for this. It will have a different styling.